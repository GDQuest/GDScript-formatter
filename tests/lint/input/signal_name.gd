#region Good

signal good_signal(var1: int, var2: String)

#endregion

#region Bad

signal BadSignal(var1: int, var2: String)
signal _bad_signal(var1: int, var2: String)
signal badSignal(var1: int, var2: String)

#endregion
