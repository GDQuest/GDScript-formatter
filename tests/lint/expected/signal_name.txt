tests/lint/input/signal_name.gd:9:signal-name:error: Signal name 'BadSignal' should be in snake_case format
tests/lint/input/signal_name.gd:10:signal-name:error: Signal name '_bad_signal' should be in snake_case format
tests/lint/input/signal_name.gd:11:signal-name:error: Signal name 'badSignal' should be in snake_case format